src/components/Notifications/TimeConfig.tsx

```diff
@@ -1,36 +1,52 @@
-
-import { Alert, Button, Card, CardBody, CardFooter, Dropdown, DropdownItem, DropdownToggle, HelperText, HelperTextItem,
-    Radio, Skeleton, Split, SplitItem, Stack, StackItem,
+import { Button, Card, CardBody, CardFooter, Dropdown, DropdownItem, DropdownToggle, HelperText, HelperTextItem,
+    Radio, Split, SplitItem, Stack, StackItem,
     Text, TextVariants, TimePicker, Title } from '@patternfly/react-core';
-import React, { useMemo } from 'react';
+import { global_spacer_lg } from '@patternfly/react-tokens';
+import React, { useEffect, useMemo } from 'react';
 import timezones from 'timezones.json';
 import { style } from 'typestyle';
 
 import { useGetTimePreference } from '../../services/Notifications/GetTimePreference';
 import { useUpdateTimePreference } from '../../services/Notifications/SaveTimePreference';
+import { LocalTime } from '../../types/Notification';
+import { useNotification } from '../../utils/AlertUtils';
 
 const dropDownClassName = style({
     width: '280px'
 });
 
+const dropDownPaddingClassName = style({
+    paddingLeft: global_spacer_lg.value
+});
+
 export const TimeConfigComponent: React.FunctionComponent = () => {
 
-    const [ radioSelect, setRadioSelect ] = React.useState(false);
+    const [ radioSelect, setRadioSelect ] = React.useState(true);
     const [ showCustomSelect, setShowCustomSelect ] = React.useState(false);
+    const [ timeSelect, setTimeSelect ] = React.useState<Partial<LocalTime>>();
 
     const getTimePreference = useGetTimePreference();
     const saveTimePreference = useUpdateTimePreference();
+    const { addSuccessNotification, addDangerNotification } = useNotification();
 
     const timePref = useMemo(() => {
-        if (getTimePreference.payload?.status === 200) {
-            return getTimePreference.payload.value;
+        if (getTimePreference.status === 200) {
+            return getTimePreference.payload?.value as string;
         }
 
         return undefined;
 
-    }, [ getTimePreference.payload?.status, getTimePreference.payload?.value ]);
+    }, [ getTimePreference.payload?.value, getTimePreference.status ]);
+
+    // Set the time preference value once we load it from the server
+    useEffect(() => {
+        if (timePref) {
+            setTimeSelect(timePref);
+        }
+    }, [ timePref ]);
 
-    const handleRadioSelect = React.useCallback(() => {
+    const handleRadioSelect = React.useCallback((time) => {
+        setTimeSelect(time);
```

Comment:
What is this `time` value? Is this the default time? where does it come from?

Reply:
this was looking to save the default time, but looking again it should be body as the prop. If someone selects the default time, i did not have a way to send what that was.

Reply:
well, that is also not right... the body is returning a boolean...

Reply:
Not needed as the value is being sent when "save" is hit.